### PR TITLE
Fix permissions and readme

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: packages/ra-auth-cognito/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "ra-auth-cognito",
-    "version": "1.0.0",
+    "version": "1.0.1",
```
Comment (slax57) [inline packages/ra-auth-cognito/package.json:3]: Maybe it's a good time to introduce a Changelog, what do you think? :innocent:

File: packages/ra-auth-cognito/Readme.md [hunk 2]
```diff
@@ -79,6 +79,7 @@ const App = () => {
            authProvider={authProvider}
            dataProvider={dataProvider}
            title="Example Admin"
+           loginPage={false} // We don't need the login page in this case
```
Comment (slax57) [inline packages/ra-auth-cognito/Readme.md:82]: I believe this is actually buggy in RA. Until the bug is fixed, we should not encourage to use that option I believe.

**Comment by fzaninotto** [comment]
Can you explain the bug in the PR description?

File: packages/ra-auth-cognito/Readme.md (hunk 2 above)
Comment (slax57) [reply packages/ra-auth-cognito/Readme.md:82]: Related issue: https://github.com/marmelab/react-admin/issues/8685

Comment (djhi) [reply packages/ra-auth-cognito/Readme.md:82]: As discussed, it's not

Comment (stripethree) [reply packages/ra-auth-cognito/Readme.md:82]: 👋🏼 I posted some questions in the Discord the other week and wanted to chime in here if that's alright.

I was chasing an issue where on logout, with a Cognito app integration that has Google as the only identity provider, the username/password form would flash before the redirection to the AWS Hosted UI. Changing this value to `false` fixed that issue. I hadn't tried that before seeing this PR.

For reference, I'm using `react-admin` 4.7.1

**Pull request merged**